tests/e2e/specs/modules/search-console/admin-bar.test.js

```diff
@@ -42,6 +42,11 @@ describe( 'Site Kit admin bar component display', () => {
 					status: 200,
 					body: JSON.stringify( mockBatchResponse[ 'modules::search-console::searchanalytics::e74216dd17533dcb67fa2d433c23467c' ] ),
 				} );
+			} else if ( request.url().match( 'google-site-kit/v1/data/' ) ) {
+				request.respond( {
+					status: 200,
+					body: JSON.stringify( mockBatchResponse ),
+				} );
```

Comment:
This was removed in a recent PR for the admin bar, but should have been kept. It's been restored in the other admin bar PR but I've added it here to for completeness.